app/Repositories/ContactRepository.php

```diff
@@ -1,24 +1,11 @@
-<?php
-
-namespace App\Repositories;
+<?php namespace App\Repositories;
 
 use App\Models\Contact;
+use Illuminate\Contracts\Pagination\LengthAwarePaginator;
 
-class ContactRepository
+interface ContactRepository
```

Comment:
Extract out an interface, as there are more than one implementations of this concept. 

Add a `store` method, so it mirrors a real repo and allows us to see when something is stored.